sa-gateway/Readme.md

```diff
@@ -0,0 +1,19 @@
+Go to folder k8s.
+
+Deploy to Kubernetes:
+
+`kubectl apply -f compiled.yaml`
```

Comment:
If kustomize is used, then it's output is piped to kubectl apply

Reply:
done